src/pocketmine/OfflinePlayer.php

```diff
@@ -43,7 +43,7 @@ class OfflinePlayer implements IPlayer, Metadatable{
 	 */
 	public function __construct(Server $server, string $name){
 		$this->server = $server;
-		$this->name = $name;
+		$this->name = strtolower($name);
```

Comment:
I was talking about the `file_exists()` check. The case of the passed name should be preserved on the property.